frontend/src/Editor/Components/Table/Table.jsx

```diff
@@ -46,6 +46,64 @@ import GenerateEachCellValue from './GenerateEachCellValue';
 import { toast } from 'react-hot-toast';
 import { Tooltip } from 'react-tooltip';
 
+// utility function
+function removeAllAddedElements(array, addedElements) {
+  for (let i = 0; i < addedElements.length; i++) {
+    const index = array.indexOf(addedElements[i]);
+    if (index !== -1) {
+      array.splice(index, 1);
+    }
+  }
+}
+function utilityToUpdateTableDetails(
```

Comment:
See if you can rename this function to an easily understandable name.

Also try to see if you can avoid mutation and use pure functions.